handler/src/main/java/io/netty/handler/ssl/SslHandler.java

```diff
@@ -538,18 +548,14 @@ public void flush(ChannelHandlerContext ctx) throws Exception {
         }
         try {
             wrap(ctx, false);
-        } catch (Throwable cause) {
-            // Fail pending writes.
-            pendingUnencryptedWrites.removeAndFailAll(cause);
```

Comment:
can you clarify why we no longer want to fail/release all pending writes in the event of an Throwable (other than `SSLException`)? Are we sure there will be no unchecked exceptions, and if there is will we potentially leak?